torch/csrc/jit/serialization/unpickler.cpp

```diff
@@ -261,9 +261,8 @@ void Unpickler::run() {
 void Unpickler::setInput(size_t memo_id) {
   AT_ASSERT(!stack_.empty());
   if (memo_id >= memo_table_.size()) {
-    memo_table_.insert(
-        memo_table_.end(), memo_id - memo_table_.size(), IValue());
-    memo_table_.push_back(stack_.back());
+    memo_table_.resize(memo_id + 1);
+    memo_table_[memo_id] = stack_.back();
   } else {
     memo_table_[memo_id] = stack_.back();
   }
```

Comment:
```suggestion
  }
  memo_table_[memo_id] = stack_.back();
```

Reply:
Will do. I have that already in a stacked diff internally, to be exported here to a new GH PR.

Reply:
Requested changes are in: #163447.